CSS/style2.css

```diff
@@ -1,21 +1,432 @@
+    font-weight: lighter;
+    font-size: 12px;
+    line-height: 20px;
+    padding: 10px;
+    margin: 10px 4px;
+    color: #fff;
+    background: rgba(255, 255, 255, 0.24);
+  }
+
+  .work-card3 {
+    width: 100%;
+    height: auto;
+    background-image: url('../Icons/workcard3-img.png');
+    background-repeat: no-repeat;
+    background-size: cover;
+    border: none;
+    padding-top: 200px;
+    padding-left: 14px;
+    padding-right: 14px;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .work-card3 h3 {
+    font-family: 'Crete Round', serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 32px;
+    line-height: 44px;
+    color: #fff;
+  }
+
+  .work-card3 p {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 100;
+    font-size: 14px;
+    line-height: 20px;
+    color: #fff;
+  }
+
+  .work-card3 ul {
+    display: flex;
+    flex-direction: row;
+    width: 70%;
+    justify-content: space-between;
+  }
+
+  .work-card3 li {
+    font-family: 'Inter', sans-serif;
+    font-weight: lighter;
+    font-size: 12px;
+    line-height: 20px;
+    padding: 10px;
+    margin: 10px 4px;
+    color: #fff;
+    background: rgba(255, 255, 255, 0.24);
+  }
+
+  .work-card4 {
+    width: 100%;
+    height: auto;
+    background-image: url('../Icons/workcard4-img.png');
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-size: 100% 100%;
+    border: none;
+    padding-top: 158px;
+    padding-left: 14px;
+    padding-right: 14px;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .work-card4 h3 {
+    font-family: 'Crete Round', serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 32px;
+    line-height: 44px;
+    color: #fff;
+  }
+
+  .work-card4 p {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 100;
+    font-size: 14px;
+    line-height: 20px;
+    color: #fff;
+  }
+
+  .work-card4 ul {
+    display: flex;
+    flex-direction: row;
+    width: 70%;
+    justify-content: space-between;
+  }
+
+  .work-card4 li {
+    font-family: 'Inter', sans-serif;
+    font-weight: lighter;
+    font-size: 12px;
+    line-height: 20px;
+    padding: 10px;
+    margin: 10px 4px;
+    color: #fff;
+    background: rgba(255, 255, 255, 0.24);
+  }
+
+  .about-me {
+    position: relative;
+  }
+
+  .about-text {
+    margin-left: 142px;
+  }
+
+  .about-text p {
+    font-size: 14px;
+    width: 34%;
+  }
+
+  .desktop-side-image2 {
+    display: block;
+    position: absolute;
+    top: 20px;
+    right: -22px;
+  }
+
+  .text-line {
+    margin: 220px 0 142px 0;
+  }
+
+  .language,
+  .framewok,
+  .skills {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .tech-competencies {
+    padding: 0 142px;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  .contact-wrap {
+    margin-left: 135px;
+    margin-right: 142px;
+    display: flex;
```

Comment:
Because of these margins and because the media queries is set early to jump to the desktop, the contact form on an 800px screen-wide looks like this:
![image](https://user-images.githubusercontent.com/83596135/160216213-743e2149-5230-4455-b724-3cdcaea0c9c2.png)

- [ ] Kindly, try to address this issue in order to have an app fully responsive for a wide variety of mobiles, tablets and desktops;

**You are doing a great job** :confetti_ball: